### PR TITLE
Add section on JetBrains IDEs

### PR DESCRIPTION
**Pull request by simeg**
Please review the markdown by first pressing the "View" button to see the rendered markdown.

I hope this text doesn't scream "I LOVE JETBRAINS" too much 😋  Any feedback is appreciated.

Is _JetBrains IDEs_ a good menu item name? It was the best I could come up with.

### CODE REVIEW

File: JetBrains IDEs/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+### Integrated Tools
+Their IDEs offer so many tools that you never have to leave it if you don't want to. It supports a number of different VCSs (Git, Subversion, Perforce etc.), a terminal window, database browser, debuggers, SSH sessions and more.
+
+### Powerful Refactoring
+Rename a method used all over your project with a few clicks, or change a method signature in the same way. With statically typed languages like Java and C the refactoring becomes even more powerful.
+
+## Useful Shortcuts
+* Search for a file -- `CMD + SHIFT + O`
+* Search for a string -- `CMD + SHIFT + F`
+* Search everything, press `SHIFT` three (3) times
```
Comment (Kyslik) [inline JetBrains IDEs/README.md]: Not sure, but on my PHPStorm installation I use `SHIFT` twice for search everywhere, maybe defaults changed? Or perhaps it differs from JetBrains product.

**Comment by sb2nov** [comment]
LGTM, I really like IntelliJ and have mostly stuck to that for all large projects.

**Comment by sb2nov** [comment]
Ahh we forgot to change the summary file. Please self merge that change.

**Comment by sb2nov** [comment]
Related to #154

File: JetBrains IDEs/README.md (hunk 1 above)
Comment (simeg) [reply JetBrains IDEs/README.md]: I tried in a few different and yes, you are right. I'll change it.

**Pull request merged**